src/Variables/DefaultVariablesProvider.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of Composer Extra Files Plugin.
+ *
+ * (c) 2017 Last Call Media, Rob Bayliss <rob@lastcallmedia.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
```

Comment:
Headers are added by PHP CS Fixer automatically.

There will be a pull request to add a step for running PHP CS Fixer in github action later.

Reply:
Good idea.

Reply:
This file is removed in the new solution.